src/integration_tests/test_cell_map_proof.py

```diff
@@ -0,0 +1,92 @@
+from typing import Final, Iterable, Tuple
+
+import pytest
+
+from pyk.cterm import CTerm
+from pyk.kast.inner import KApply, KInner, KSequence, KToken, KVariable, build_assoc
+from pyk.kast.manip import get_cell
+from pyk.ktool import KProve
+
+from .utils import KProveTest
+
+State = Tuple[str, str, Iterable[Tuple[str, str]]]
```

Comment:
After

```python3
from typing import NamedTuple
```

you can

```suggestion
class State(NamedTuple):
    pgm: str
    active_accounts: str
    accounts: Iterable[Tuple[str, str]]
    
```